src/lib/seam/components/SupportedDeviceTable/use-device-provider.ts

```diff
@@ -0,0 +1,22 @@
+import type { DeviceProvider } from 'seamapi'
+
+import { useDeviceProviders } from 'lib/seam/devices/use-device-providers.js'
+
+export function useDeviceProvider(brand: string): DeviceProvider {
```

Comment:
I was going to suggest we return null or isLoading while the hook is loading, but maybe the behavior that the provider will show as unknown until it loads or in case it fails is better UX since things will still partially render if this is loading or fails.